### DISCUSSION TITLE
Hi Everyone, Im having huge problems with JOYPRO could any help please

### DISCUSSION THREAD

**Discussion by HoovaStank** [community/community#34258] [Other Feature Feedback, Questions, & Ideas]
hey guys, first off great work Holdi601 on the JOYPRO its absolutely brilliant, anyways, originally when I downloaded and followed the step it worked great, then I got scamed by by a web page scam and went as far was letting the guy access to my computer, I didnt know, its never happen to me, I feel like an idoit, but at the same time, how could he, its just not right, people have family and identy theft or fraud is so wrong, I would rather be stabbed..   I am pretty good with computer learning alot of the mechanics however this I can not seem to fix and def need help, I play DCS World and I can not get my profiles to work in game with DCS control menu, I would love to have joypro back and work, o sorry I left out the fact that after this guy got on my pc, i ended up wiping it losing the JOYPRO that was working for me.  as of today ive download and unpacked 14 versions with no luck...  hopefully someone is willing...   THANK YOU @Holdi601 for this your a genius..

**Comment by Holdi601** [comment]
Hello. Im sorry to hear that you have become a victim of a scam. These people are truely scum. Lets get to joypro, have you checked that you have .Net 4.8 minimum installed? have you installed slimdx runtime x64? 
Is the icon in tray and just not full screen or is it instantly gone. Is there any exception files in the root folder?

**Reply by HoovaStank** [reply]
Hey Holdi601, finally ive found time to resolve this, so I just got a new stick and grip, logitech x52 space/flight HOTAS, Ive tried many times to get joyP  working and ever since the first time, ive never again had it go.  If you could please help me I would greatly apprciate it.  Maybe we could meet on discord and I could screen share..   (Huge Fan of your work)    hope to hear from you soon.  Hoova

**Reply by Holdi601** [reply]
Have you checked for dependencies like also slim dx runtime? 
Sure discord is always an option. My community discord for joypro is listed in joypros description. https://discord.gg/SeCTXJHhJf

**Reply by HoovaStank** [reply]
Im not so sure what that is...  I will check out the link, if you do have a couple minutes free I would be happy to have you just boss through it on a shared connections.  (Your Respected in the community)

**Comment by HoovaStank** [comment]
Hey Holdi,

A buddy of mine has my computer, he is deleting the partitions and reinstalling everything..   thank you so much ,an, I will contact you once I have it back and dcsw install right before I'm going to head over to your page.

Thanks again, I hope this works for you

Jay

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
